### PR TITLE
[docs] Describe screen recording permissions

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
fixes https://github.com/DevExpress/testcafe/issues/4687

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/installing-testcafe.md [hunk 1]
```diff
@@ -81,3 +81,13 @@ Use `npx testcafe` if you need to quickly run TestCafe from a directory outside
...
+
+## Screen Recording Permission
+
+TestCafe requires screen recording permission on macOS (v10.15 Catalina and newer) to perform test actions, take screenshots and record videos. When TestCafe starts the first time, macOS asks you to allow screen recording for *TestCafe Browser Tools*.
+
+![Screen Recording Permissions](../../images/recording-permissions.png)
+
+Click **Open System Preferences** and check **TestCafe Browser Tools** to grant the permission.
+
+When you update TestCafe, macOS resets the security permissions and repeats the request the next time you launch TestCafe.
```
Comment (anthophobiac) [inline docs/articles/documentation/using-testcafe/installing-testcafe.md]: When you update TestCafe, macOS resets its security permissions. The next time you run tests, the system repeats the request.

File: docs/articles/documentation/using-testcafe/installing-testcafe.md [hunk 2]
```diff
@@ -81,3 +81,13 @@ Use `npx testcafe` if you need to quickly run TestCafe from a directory outside
...
+
+## Screen Recording Permission
+
+TestCafe requires screen recording permission on macOS (v10.15 Catalina and newer) to perform test actions, take screenshots and record videos. When TestCafe starts the first time, macOS asks you to allow screen recording for *TestCafe Browser Tools*.
+
+![Screen Recording Permissions](../../images/recording-permissions.png)
+
+Click **Open System Preferences** and check **TestCafe Browser Tools** to grant the permission.
+
+When you update TestCafe, macOS may reset its security permissions (it depends on which TestCafe components are updated). In this case, the system repeats the request the next time you run tests.
```
Comment (AndreyBelym) [inline docs/articles/documentation/using-testcafe/installing-testcafe.md]: The problem was that the system kept spawning consent dialogs despite the mark in the "Privacy & Security" Settings was ticked. Users were confused and couldn't understand that they had to clear and re-tick the mark.

Comment (AndreyBelym) [reply docs/articles/documentation/using-testcafe/installing-testcafe.md]: Also, I feel that attaching a screenshot of the Privacy & Security panel is required:

img

Comment (AndreyBelym) [reply docs/articles/documentation/using-testcafe/installing-testcafe.md]: ping @VasilyStrelyaev

File: docs/articles/documentation/using-testcafe/installing-testcafe.md [hunk 3]
```diff
@@ -81,3 +81,15 @@ Use `npx testcafe` if you need to quickly run TestCafe from a directory outside
...
 npx testcafe chrome tests/
 ```
+
+## Screen Recording Permission
+
+TestCafe requires screen recording permission on macOS (v10.15 Catalina and newer) to perform test actions, take screenshots and record videos. When TestCafe starts the first time, macOS asks you to allow screen recording for *TestCafe Browser Tools*.
+
+![Screen Recording Permissions](../../images/recording-permissions.png)
+
+Click **Open System Preferences** and check **TestCafe Browser Tools** to grant the permission.
```
Comment (arminal) [inline docs/articles/documentation/using-testcafe/installing-testcafe.md]:
```suggestion
Click **Open System Preferences** and check **TestCafe Browser Tools** to grant permission.
```

**Pull request merged**